tasks/sep/fp-recovery/002-fallback-permissioned-game/README.md

```diff
@@ -0,0 +1,54 @@
+# Deputy Guardian - Fall back to `PermissionedDisputeGame`
+
+This batch udates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`. This action requires all in-progress withdrawals to be re-proven against a new `PermissionedDisputeGame` that was created after this update occurs.
+
+The batch will be executed on chain ID `11155111`, and contains `1` transactions.
+
+## Tx #1: Update `respectedGameType` in the `OptimismPortal`
+
+Updates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`, enabling permissioned proposals and challenging.
+
+**Function Signature:** `setRespectedGameType(address,uint32)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xa1155ed900000000000000000000000016fc5058f25648194471939df75cf27a2fdc48bc0000000000000000000000000000000000000000000000000000000000000001`
+
+### Inputs
+
+**\_gameType:** `1` (`PERMISSIONED_CANNON`)
+
+**\_portal:** `0x16Fc5058F25648194471939df75CF27A2fdC48BC`
+
+## Preparing the Operation
+
+1. Collect signatures for the action.
+
+1. Execute the action.
+
+## Signing and execution
+
+Please see the signing and execution instructions in [SINGLE.md](../../../../SINGLE.md).
```

Comment:
Similar to the prev comment, these two sections are redundant. Let's clean it up please.